nitrite_test.go

```diff
@@ -7,79 +7,141 @@ import (
 	"testing"
 	"time"
 
-	"github.com/blocky/nitrite"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	"github.com/blocky/nitrite"
 )
 
-// An easy way to create a new nitro attestation is to use delphi-cli.
-// Request an attestation over some endpoint and copy the resulting
-// "enclave_attestation" string from the output---it should be a
-// base64 encoded string
-//
-//go:embed testdata/nitro_attestation.txt
-var nitroAttestString string
+// The canonical way to regenerate attestations is to request an attestation
+// using hf/nsm/Send() and write the resulting bytes to a file as a base64
+// string.
 
-func TestNitrite_Verify(t *testing.T) {
-	attestBytes, err := base64.StdEncoding.DecodeString(nitroAttestString)
-	require.NoError(t, err)
+//go:embed testdata/nitro_attestation.b64
+var attestationB64 string
+var attestationTime = time.Date(2024, time.September, 7, 14, 37, 39, 545000000, time.UTC)
 
-	doc, err := nitrite.NewDocumentFromCosePayloadBytes(attestBytes)
+//go:embed testdata/nitro_attestation_debug.b64
+var debugAttestationB64 string
+
+func TestNitrite_Verify(t *testing.T) {
+	attestation, err := base64.StdEncoding.DecodeString(attestationB64)
 	require.NoError(t, err)
 
 	roots := x509.NewCertPool()
 	ok := roots.AppendCertsFromPEM(nitrite.AWSNitroEnclavesCertPEM)
 	require.True(t, ok)
 
-	t.Run("happy path", func(t *testing.T) {
-		opts := nitrite.VerifyOptions{
-			CurrentTime: doc.CreatedAt(),
-			Roots:       roots,
-		}
+	happyPathTests := []struct {
+		name           string
+		attestationB64 string
+	}{
+		{
+			"regular attestation",
```

Comment:
you may want to label these.  Since both are strings, dropping the fields can be error prone.

Reply:
done